core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/listofobjectstolistofstring/User.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.listofobjectstolistofstring;
+
+public class User {
+    private final String userName;
+    private final String firstName;
+    private final String lastName;
+
+    public User(String userName, String firstName, String lastName) {
+        this.userName = userName;
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    @Override
+    public String toString() {
+        return "User{" +
+                "userName='" + userName + '\'' +
+                ", firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
+                '}';
```

Comment:
Let's follow the pattern from the `Node` class for this `toString()`

```suggestion
        return "User (" +
                "userName='" + userName + '\'' +
                ", firstName='" + firstName + '\'' +
                ", lastName='" + lastName + '\'' +
                ')';
```